corehq/apps/app_manager/tasks.py

```diff
@@ -3,13 +3,17 @@
 from django.utils.translation import ugettext as _
 
 from celery.task import task
+from celery.utils.log import get_task_logger
 
 from corehq.apps.app_manager.dbaccessors import get_app, get_latest_build_id, get_auto_generated_built_apps
 from corehq.apps.app_manager.exceptions import SavedAppBuildException
 from corehq.apps.users.models import CommCareUser
 from corehq.util.decorators import serial_task
 
 
+logger = get_task_logger('app_manager')
```

Comment:
i've been bitten before by not knowing where a log will go to with this kind of thing: would it show up in the `www/<env>/log/celery_<task>` log?

Reply:
I'm not familiar with `get_task_logger`. The conventional way to get a logger in Python is
```py
logger = logging.getLogger(__name__)
```
However, celery may have special requirements that are not met by that method? It smells a little bit to be passing in `'app_manager'` rather than `__name__`, but maybe that's our convention?

Reply:
Updated to use `__name__` - we use both styles, but I agree hard coded strings are usually a bad idea.

Reply:
and I believe this will show up in `www/<env>/log/celery_<queue>_<index>.log`